src/test/fuzz/torcontrol.cpp

```diff
@@ -1,10 +1,11 @@
-// Copyright (c) 2020 The Bitcoin Core developers
+// Copyright (c) 2020-2021 The Bitcoin Core developers
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 #include <test/fuzz/FuzzedDataProvider.h>
 #include <test/fuzz/fuzz.h>
 #include <test/fuzz/util.h>
+#include <test/util/setup_common.h>
```

Comment:
Which symbol requires this include?

Reply:
Probably none in this file, but adding it in this commit will avoid a rebase in https://github.com/bitcoin/bitcoin/pull/21003#issuecomment-790405321

Reply:
Does #21003 touch `src/test/fuzz/torcontrol.cpp` ?

Reply:
No, but the `MakeNoLogFileContext` symbol is moved to a different file (via the indirect dependencies)